f2/src/components/ErrorSummary/ErrorSummary.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'
+import ThemeDecorator from '../../../.storybook/themeDecorator'
+import FormDecorator from '../../../.storybook/formDecorator'
+import { withKnobs, boolean, text } from '@storybook/addon-knobs'
+import { Field } from '../Field'
+import { Input } from '../input'
+import { TextArea } from '../text-area'
+import { ErrorSummary } from '.'
+import { Stack } from '@chakra-ui/core'
+import { Trans } from '@lingui/react'
```

Comment:
import trans from lingui/react, not lingui/macro. Only in storybook. [See this reference](https://lingui.js.org/tutorials/react.html#macros-vs-components)